angular-rails-app/db/migrate/20170225185334_init_schema.rb

```diff
@@ -123,13 +123,13 @@ def up
       t.string   "zipcode",             limit: 255
       t.integer  "total_units",         limit: 4
       t.string   "bin",                 limit: 255
-      t.boolean  "rent_stabilized"
+      t.integer  "rent_stabilized",     limit: 4
```

Comment:
Ohh, have you just been changing the initial migration? And using db:schema:load?

That might explain why you've been seeing some wonkiness when trying to make changes. Lindsey and I have been using migrations to make changes. 

I think we should probably be using migrations, since `db:schema:load` can destroy data in production. [Here's a pretty good rundown](http://stackoverflow.com/questions/5905287/rake-dbschemaload-vs-migrations)